src/providers/WorkflowCore.Providers.Azure/Services/CosmosDbPersistenceProvider.cs

```diff
@@ -19,19 +19,19 @@ public class CosmosDbPersistenceProvider : IPersistenceProvider
 
         private ICosmosDbProvisioner _provisioner;
         private string _dbId;
-        private CosmosClient _client;
+        private ICosmosDbClient _client;
```

Comment:
Should this be named to indicate it's a client factory rather than a client itself?

Reply:
Make sense. Thanks, a lot.